oidc-controller/api/db/session.py

```diff
@@ -23,6 +25,17 @@ async def init_db():
     auth_session.create_index([("pres_exch_id", ASCENDING)], unique=True)
     auth_session.create_index([("pyop_auth_code", ASCENDING)], unique=True)
 
+    expire_time: int = settings.CONTROLLER_PRESENTATION_EXPIRE_TIME + settings.CONTROLLER_PRESENTATION_BUFFER_TIME
+
+    for k, v in [("expired_ttl", AuthSessionState.EXPIRED),
+                 ("failed_ttl", AuthSessionState.FAILED),
+                 ("abandoned_ttl", AuthSessionState.ABANDONED)]:
```

Comment:
Is there a way we could provide this in a configuration file so that behaviour could be customized per-deployment? In that case, how would we ensure that indexes that have been removed from the config are removed from the mongodb TTL settings as well?

Reply:
Likely yes. Let me look through the documentation on mongoDB's JS interface

Reply:
I guess you probably mean a dedicated config file. That is also possible (and probably preferable)

Reply:
I have moved this to a json config file